### PR TITLE
Change ui scheme and bad avatars

### PR DESCRIPTION
**Pull request by Kevbyte**

### CODE REVIEW

File: client/assets/css/style.css [hunk 1]
```diff
@@ -1264,4 +1265,8 @@ button:focus {
 
 .collab {
   overflow: auto;
+}
+
+.sel {
+  width:214.4px;
 }
```
Comment (codydaig) [inline client/assets/css/style.css]: New line

**Comment by AndrewKishino** [comment]
LGTM

**Comment by lina** [comment]
lgtm

**Comment by codydaig** [comment]
LGTM

**Pull request merged**